cells/lib/ops/revisePackage.nix

```diff
@@ -0,0 +1,34 @@
+{
+  inputs,
+  cell,
+}: let
+  inherit (inputs) nixpkgs;
+  l = nixpkgs.lib // builtins;
+in
+  /*
+  For use with `revise` and `reviseOCI` to build containers in a mono-repo style
+  environment where the source code is contained in the same repository as the std code,
+  specifically so that one may detect meaningful changes to the image via its tag in the
+  special case where the package's output includes the revision of the source code (e.g. for
+  displaying the version to the user).
+
+  Without special processing, this kind of package would cause the OCI image tag to change
+  on each new revision whether the actual contents of the image changed or not. Combined
+  with `std.incl`, one may have a very strong indicator for when the contents of the image
+  actually includes meaningful changes which avoids flooding the remote registry with superflous
+  copies.
```

Comment:
We could explain how this applies to packages here ☺️

Reply:
I've struggled to explain this from the beginning, so if you have any suggestions I'm all for it :sweat_smile: